src/main/java/it/pagopa/interop/probing/caller/config/kms/security/JwtBuilder.java

```diff
@@ -0,0 +1,80 @@
+package it.pagopa.interop.probing.caller.config.kms.security;
+
+import java.nio.ByteBuffer;
+import java.util.Base64;
+import java.util.UUID;
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.model.SignRequest;
+import com.amazonaws.services.kms.model.SignResult;
+import com.amazonaws.services.kms.model.SigningAlgorithmSpec;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.Payload;
+import com.nimbusds.jose.shaded.json.JSONObject;
+import com.nimbusds.jwt.JWTClaimsSet;
+
+@Configuration
+public class JwtBuilder {
+
+  @Autowired
+  private AWSKMS kms;
+
+  @Value("${jwt.payload.expire-time-in-seconds}")
+  private Integer exp;
+
+  @Value("${jwt.payload.issuer}")
+  private String issuer;
+
+  @Value("${jwt.payload.subject}")
+  private String subject;
+
+  @Value("${jwt.payload.kid-kms}")
+  private String kid;
+
+  public String buildJWT(String[] audience) {
+
+    String token = createToken(audience);
+    SignRequest signReq = new SignRequest().withKeyId(kid)
+        .withSigningAlgorithm(SigningAlgorithmSpec.RSASSA_PKCS1_V1_5_SHA_256)
```

Comment:
since it is used in multiple places, I suggest to move `SigningAlgorithmSpec.RSASSA_PKCS1_V1_5_SHA_256` in a local constant

Reply:
ok, done